public/index.html

```diff
@@ -19,7 +20,7 @@
       Notice the use of %PUBLIC_URL% in the tags above.
       It will be replaced with the URL of the `public` folder during the build.
       Only files inside the `public` folder can be referenced from the HTML.
-
+      
```

Comment:
Generally if there is no change of note in a file, like what we have here, it's best to keep the file unchanged in the diff. This limits the amount of commits in this file's history, making it easier in the future to understand the changes made.

The easiest way to undo these changes is to run `git checkout origin/master public/index.html` from your local repo. This will check out the version in `master`. Bonus points if you squash your commits to remove changes from this PR's history altogether, but not a requirement.

Reply:
Did a short dive into squashing commits. I had already pushed the other commits. Would love to touch base and get some clarity on how I should have done this.